ui/app/common/auth/authentication.js

```diff
@@ -56,13 +56,25 @@ angular.module('authentication')
                     deferrable.resolve(response.data);
                 }, function (response) {
                     if (response.status == 401) {
-                        deferrable.reject('LOGIN_LABEL_WRONG_OTP_MESSAGE_KEY');
+                        getAuthFromServer(username, password, otp).then(function (response) {
```

Comment:
Making authentication call API again because invalid session at client side after **http DELETE** API call for `v1/session` ([openmrs/openmrs-module-webservices.rest@cc6c1ab#diff-6a9196b7867cc6b86b108db49b6dfffad7f2fc49a0da12cf40169e31f1564cbaR128](https://github.com/openmrs/openmrs-module-webservices.rest/commit/cc6c1abaf9eab4620d8c66ad9d7a22a924ca21fd#diff-6a9196b7867cc6b86b108db49b6dfffad7f2fc49a0da12cf40169e31f1564cbaR128))